$MirOS$

	XXX hand-edited due to no newline at end of file
	add all dependent libs here

--- SDL_Pango.pc.in.orig	Tue Sep  7 17:30:33 2004
+++ SDL_Pango.pc.in	Sat Nov  4 00:33:31 2006
@@ -7,4 +7,4 @@ Name: SDL_Pango
 Description: SDL library for internationalized text rendering
 Version: @VERSION@
 Requires: pango
-Libs: -L${libdir} -lSDL_Pango
+Libs: -L${libdir} -lSDL_Pango @LIBS@
